CHANGELOG.md

```diff
@@ -68,17 +68,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Middleware support (for e.g. for centrally handling authorization).
 
 ## [1.1.0] - 2019-05-30
+### Added
+- Travis builds for tags.
 
 ## [1.0.6] - 2019-05-19
```

Comment:
In general, looks correct, but missed two user-visible commits in README:
- https://github.com/tarantool/http/commit/da7121cedf2614eebcd39cdc6a0c48ef75fde96b
- https://github.com/tarantool/http/commit/518d62c47fd1c7e414abe0ad86075fbbe174c95c
Changes in 1.0.6: https://github.com/tarantool/http/compare/1.0.5...1.0.6